README.md

```diff
@@ -211,32 +211,16 @@ RxJavaPlugins.setErrorHandler { throwable ->
 #### Which permissions are needed?
 **Android**
 
-For android the library uses the following permissions, depending on the SDK level:
-
-Up to SDK 30 (Android 11):
-* ACCESS_FINE_LOCATION : this permission is needed because old Nexus devices need location services in order to provide reliable scan results
-* BLUETOOTH : allows apps to connect to a paired Bluetooth device
-* BLUETOOTH_ADMIN: allows apps to discover and pair Bluetooth devices
-* BLUETOOTH_SCAN: add this permission with `tools:node="remove"` to remove it from the merged manifest.
-``` 
-<uses-permission
-            android:name="android.permission.BLUETOOTH_SCAN"
-            android:usesPermissionFlags="neverForLocation" tools:node="remove" />
+You need to add the following permissions to your AndroidManifest.xml file:
+```xml
+<uses-permission android:name="android.permission.BLUETOOTH_SCAN" android:usesPermissionFlags="neverForLocation" />
```

Comment:
Thanks for your contribution, BLUETOOTH_SCAN was added only in API 31, why did you remove distinguish between API 31,30?

Reply:
That's from the official documentation: https://developer.android.com/guide/topics/connectivity/bluetooth/permissions